homeassistant/components/husqvarna_automower/binary_sensor.py

```diff
@@ -81,3 +100,37 @@ def __init__(
     def is_on(self) -> bool:
         """Return the state of the binary sensor."""
         return self.entity_description.value_fn(self.mower_attributes)
+
+    async def async_added_to_hass(self) -> None:
+        """Raise issue when entity is registered and was not disabled."""
+        if TYPE_CHECKING:
+            assert self.unique_id
+        if entity_id := er.async_get(self.hass).async_get_entity_id(
+            BINARY_SENSOR_DOMAIN, DOMAIN, self.unique_id
+        ):
+            if (
+                self.enabled
+                and self.entity_description.key
+                in AutomowerBinarySensorEntityTypes.RETURNING_TO_DOCK
```

Comment:
Don't use `in` to compare a single string when we want to compare the whole string. Use `==`.